boxplot

dke-chan opened the pull request:
trimPlot() can somewhat handle the boxplot function now.

dke-chan committed:
boxplot

trimPlot() can somewhat handle the boxplot function now.

dke-chan committed:
trimPlot() update

Now works with boxplot(horizontal = TRUE) and some minor improvements with cooks20x().

dke-chan committed:
pairs20x

Added an exclusion for pairs20x()... Eventually will need to tidy up the code rather than fixing "leaks" as I go.

TheGreatGospel commented:
Now handles boxplot(horizontal = FALSE), boxplot(horizontal = TRUE), and pairs20x().

dke-chan committed:
slimSummary()

Just a small patchwork fix to have slimSummary work with glms. Probably needs smarter arguments to make it more efficient to use.

dke-chan committed:
slimSummary()

Adjusted the function arguments to be smarter to differentiate between lms and glms.

dke-chan committed:
trimPlot()

Kind of works with barplot and with glm objects.

dke-chan committed:
trimPlot()

bugfix to pick up glms...

TheGreatGospel commented:
Just a heads up that we need to merge this into the master branch.
